### DISCUSSION TITLE
Error loading wallet after update

### DISCUSSION THREAD

**Discussion by marthhan** [iotaledger/firefly#7641] [Help & Support] [answered]
Hey. I'm having an issue after updating to newest version (2.0.2) today - when opening wallet I get the following error "local time <number> doesnt match the time of the latest milestone timestamp: <different number>"

I've tried restoring to a previous stronghold backup, and I've tried restoring using the firefly passphrase - same result.

Please advise?

**Comment by JannemanDev** [comment, marked as answer]
Your time/clock is not set correctly. On Windows you can sync with internet time, click on your clock in taskbar.

**Reply by begonaalvarezd** [reply]
Closing this discussion because fixing the OS time solves the issue, @marthhan please reopen if the issue persists

**Reply by marthhan** [reply]
sorry for such a delayed "mark as answer". 

but yeah, strange - my clock was correct and time zone correct. but clicking sync did the trick. 
Maybe this is due to last time sync was so long ago, regardless of time and date was correctly set?

**Reply by begonaalvarezd** [reply]
afaik in most OS you can set it to automatic so its always in sync
